api_guard/dist/types/index.d.ts

```diff
@@ -132,7 +132,7 @@ export declare class ConnectableObservable<T> extends Observable<T> {
     refCount(): Observable<T>;
 }
 
-export declare type Cons<X, Y extends any[]> = ((arg: X, ...rest: Y) => any) extends (...args: infer U) => any ? U : never;
+export declare type Cons<X, Y extends readonly any[]> = ((arg: X, ...rest: Y) => any) extends (...args: infer U) => any ? U : never;
```

Comment:
I'll be honest, the name `Cons` confuses me when I see it. It would be nice if it was called like `PrependTuple` or `UnshiftTuple` or something? (We don't have to do that in this PR, obviously, I just don't understand the name `Cons`)